lasy/optical_elements/polynomial_spectral_phase.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+
+from .optical_element import OpticalElement
+
+
+class PolynomialSpectralPhase(OpticalElement):
+    r"""
+    Class for an optical element with added spectral phase.
+
+    The amplitude multiplier corresponds to:
+
+    .. math::
+
+        T(\omega) = \exp(i(\phi(\omega)))
+
+    where :math:`\phi(\omega)` is the spectral phase given by:
+
+    .. math::
+
+        \phi(\omega) = \frac{\text{GDD}}{2!} (\omega - \omega_0)^2 + \frac{\text{TOD}}{3!} (\omega - \omega_0)^3 + \frac{\text{FOD}}{4!} (\omega - \omega_0)^4
+
+    The other parameters in this formula are defined below.
+
+    Parameters
+    ----------
+    gdd : float (in s^2)
+        Group Delay Dispersion.
```

Comment:
Suggestion on expending the documentation, so that the sign convention is clear.
```suggestion
        Group Delay Dispersion. `GDD > 0` corresponds to a positive chirp, i.e. low-frequency part of the spectrum arriving earlier than the high-frequency part of the spectrum.
```